scholly/base/src/layouts/layout1/components/layout/Header.js

```diff
@@ -16,8 +16,7 @@ const Header = () => {
       <Navbar
         collapseOnSelect
         expand="md"
-        bg="light"
-        variant="light"
+        variant=""
```

Comment:
Might be dumb: why do we need `variant=""` when there's no value in it?

Reply:
@jfan0010 Some of the components automatically assign a default variant (light or primary) if I don't have this (and that overwrites the colours I've assigned in CSS).
Navbar does this but some others don't for some reason (in those cases it's fine to delete the variant entirely).